File: service-company-b/model/product.py
class Product:
    def __init__(self, idprod, nameprod, quantityprod, priceprod, companyprod):
        self.idprod = idprod
        self.nameprod = nameprod
        self.quantityprod = quantityprod
        self.priceprod = priceprod
        self.companyprod = companyprod

    def __str__(self) -> str:
        return f"Product(id={self.idprod}, name={self.nameprod}, quantity={self.quantityprod}, price={self.priceprod}, company={self.companyprod})"

File: service-company-c/demo3/service-company-c/model/Product.cs
public class Product
{
    public int id;
    public string name;
    public int quantity;
    public double price;
    public string company;

    public Product(int id, string name, int quantity, double price, string company)
    {
        this.id = id;
        this.name = name;
        this.quantity = quantity;
        this.price = price;
        this.company = company;
    }
}
